lib/workers/repository/dependency-dashboard.ts

```diff
@@ -13,11 +13,43 @@ import { PackageFiles } from './package-files';
 interface DependencyDashboard {
   dependencyDashboardChecks: Record<string, string>;
   dependencyDashboardRebaseAllOpen: boolean;
+  dependencyDashboardAllPending: boolean;
+  dependencyDashboardAllRateLimited: boolean;
+}
+
+function checkApproveAllPendingPR(issueBody: string): string {
+  const checkApproveAllPendingPR =
+    ' - \\[x\\] <!-- approve-all-pending-prs -->';
+  let newIssueBody = issueBody;
+  if (regEx(checkApproveAllPendingPR).test(issueBody)) {
+    const checkPending = regEx(/ - \[ ] <!-- approve-branch=/gm);
```

Comment:
```suggestion
    const checkPending = regEx(/ - \[ ] <!-- approve-branch=/g);
```
i think you can exclude the `m` flag
m flag is used when you want to match the start of a line ^ and the end of the line $ on a multi line input.

Reply:
you are right